options/components/webBottomContent/WebBottomContent.jsx

```diff
@@ -4,31 +4,64 @@ import { WebSearchContext } from "../../context/WebSearchContext";
 
 export default function WebBottomContent() {
   return (
-    <div className="w-full h-[calc(100vh-200px)] overflow-hidden max-w-5xl mx-auto my-0">
+    <div className="w-full h-[calc(100vh-200px)] overflow-hidden lg:max-w-5xl lg:px-0 md:max-w-screen-md sm:max-w-screen-sm px-2 mx-auto my-0">
       <WebUrlNewList />
     </div>
   );
 }
 
 function WebUrlNewList() {
-  const { bookmarkList } = useContext(WebSearchContext);
+  const { bookmarkList, hasSearchResult } = useContext(WebSearchContext);
 
+  function faviconURL(u) {
+    const url = new URL(chrome.runtime.getURL("/_favicon/"));
+    url.searchParams.set("pageUrl", u);
+    url.searchParams.set("size", "32");
+    return url.toString();
+  }
+  console.log(hasSearchResult);
   return (
-    <div className="w-full h-[calc(100vh-200px)] overflow-y-auto">
+    <div className="w-full overflow-y-auto grid lg:grid-cols-3 md:grid-cols-2 sm:grid-cols-2 gap-4">
       {bookmarkList.map((url, index) => (
         <div
-          className="w-full flex"
+          className="w-full min-h-10 flex bg-white rounded-lg p-3 hover:bg-gray-300 hover:shadow-lg duration-100"
           key={index}
         >
           <a
-            className="w-full max-w-[calc(100%-10px)] flex-grow overflow-hidden text-ellipsis whitespace-nowrap"
+            className="w-full max-w-[calc(100%-0px)] flex-grow overflow-hidden text-ellipsis whitespace-nowrap font-bold"
             href={`${url.url}`}
             target="_blank"
           >
+            <img
+              className="mr-2 inline-block w-3 h-3"
+              src={faviconURL(url.url)}
+            />
             {url.title}
+            {hasSearchResult ? <HighlightKeyword url={url} /> : ""}
           </a>
         </div>
       ))}
     </div>
   );
 }
+
+function HighlightKeyword({ url }) {
+  const { keyword } = useContext(WebSearchContext);
+  console.log(keyword);
+
+  return (
+    <div className="mt-3 pt-3 border-t font-normal w-full max-w-[calc(100%-0px)] overflow-hidden text-ellipsis whitespace-nowrap">
+      {url.urlText.includes(keyword) ? (
+        <>
+          {url.urlText.split(keyword)[0]}
+          <span className="bg-blue-800 rounded-lg px-1 inline-block text-white mx-px">
+            {keyword}
+          </span>
+          {url.urlText.split(keyword)[1]}
+        </>
+      ) : (
+        ""
+      )}
+    </div>
```

Comment:
```suggestion
    <div className="mt-3 pt-3 border-t font-normal w-full max-w-[calc(100%-0px)] overflow-hidden text-ellipsis whitespace-nowrap">
          {url.urlText.split(keyword)[0]}
          <span className="bg-blue-800 rounded-lg px-1 inline-block text-white mx-px">
            {keyword}
          </span>
          {url.urlText.split(keyword)[1]}
    </div>
```
urlText에 keyword가 포함되지 않을 경우를 처리하기 위해 삼항연산자를 활용하여 처리한 부분은 좋은 것 같습니다. 그런데 urlText는 서버에서 항상 keyword가 포함한 데이터로 처리해서 넘겨주기 때문에 includes 함수로 처리하는 부분과 삼항 연산자는 생략해도 좋을 것 같습니다. 또한 하나의 div 태그안에 작성되어 있기 때문에 Fragment도 생략 가능할 것 같습니다. 감사합니다!